### PR TITLE
removed script error in browser

### PR DESCRIPTION
**Pull request by catevita**
By analyzing the index.html script declaration and the browser output of points_web_viewer, 
there was an error during a call to main.
Removing the lines in index.html the functionality remains the same.
It looks like there was a merge error...
..I am complete newbie in Typescript though, could you please look at this line changes?

ptal! @dudv 
fyi @pdubroy @feuerste

### CODE REVIEW

File: octree_web_viewer/client/index.html [hunk 1]
```diff
@@ -14,17 +14,18 @@
...
 <!doctype html>
+
 <head>
 </head>
+
 <body>
   <div id="renderArea" style="position: absolute; width: 100%; height: 100%; overflow: hidden;"></div>
 
   <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.5.1/dat.gui.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/r82/three.min.js"></script>
```
Comment (dudv) [inline octree_web_viewer/client/index.html]: The version of three.js is wrong here, could you please change it to point to the latest?

Comment (dudv) [reply octree_web_viewer/client/index.html]: As discussed offline, we can remove those 2 lines altogether. Thanks for bringing this up.

File: octree_web_viewer/client/index.html [hunk 2]
```diff
@@ -14,17 +14,19 @@
...
+
 <body>
   <div id="renderArea" style="position: absolute; width: 100%; height: 100%; overflow: hidden;"></div>
 
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.5.1/dat.gui.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/r82/three.min.js"></script>
-  <script data-main="app_bundle" src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.2.0/require.min.js"></script>
+  <!--script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.7.2/dat.gui.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/0.101.1/three.min.js"></script>
+  -->
```
Comment (feuerste) [inline octree_web_viewer/client/index.html]: Can you please completely delete the lines instead of commenting them out?

Comment (catevita) [reply octree_web_viewer/client/index.html]: done!

**Comment by catevita** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user catevita. Merge queue now has a length of 1.

**Pull request merged**